src/ui/actions/session.ts

```diff
@@ -163,6 +163,8 @@ export function createSocket(
       }
 
       const experimentalSettings: ExperimentalSettings = {
+        // Uncomment this to get a new session every time you refresh
+        // controllerKey: String(Math.floor(Math.random() * 10e8)),
```

Comment:
I wonder if it would be worth adding a URL flag that trips this? e.g. `?forceNewSession` or something?

Reply:
Yeah, I have also toyed with this, but never *quite* found the right way to do it, the URL flag seems like a reasonable way to do it, though I might leave it off of this PR just because it's big enough as is, especially in combination with #7732